src/test/resources/assets/forgedebugmodelloaderregistry/blockstates/CustomModelBlock.json

```diff
@@ -9,9 +9,9 @@
      },
     "variants": {
         "normal": [{
-            "transform": {
+            /*"transform": {
                 "rotation": {"y": 45}
-            }
+            }*/
```

Comment:
that statement made the model always be rotated to 45 degrees, and i wanted to show that the rotation based on the PropertyDirection was working properly without that extra 45 degrees